wave_vision/tests/matcher_tests/brute_force_tests.cpp

```diff
@@ -327,4 +347,46 @@ TEST(BFTests, DISABLED_LMEDS) {
 
     cv::waitKey(0);
 }
+
+TEST(BFTests, DISABLED_SequenceMatches) {
```

Comment:
Okay, fixed. Created `wave_vision_viz_tests`, and all visualization tests have been moved into that directory. As more modules are developed they can be added there.